### PR TITLE
Add elevation prop for Card component

### PR DESCRIPTION
**Pull request by EverWinter23**
Elevation props inspired by Material UI. Elevation prop values ranging from 1-24.

- With elevation=5
elevation5
- With elevation=15
elevation15
- With elevation=24
elevation24

### CODE REVIEW

**Comment by EverWinter23** [comment]
@anitnilay20 If the approach is sound we can start adding the elevation prop to other components.

**Comment by EverWinter23** [comment]
@anitnilay20 Updated! Chelsea won 3-1 :frowning_face:

File: src/components/Card/Card.tsx [hunk 1]
```diff
@@ -21,16 +22,20 @@ export const Card: React.FunctionComponent<CardProps> = (props) => {
...
-  stylesheet({
+const style = (theme: Theme) => {
+  return stylesheet({
     container: {
       boxShadow: shadow('2X', theme),
       background: theme.background,
       boxSizing: 'border-box',
       padding: '16px',
     },
+    elevation: {},
```
Comment (anitnilay20) [inline src/components/Card/Card.tsx:33]: do we need this?

**Review by anitnilay20 — CHANGES_REQUESTED** [review]

File: src/components/Card/Card.tsx (hunk 1 above)
Comment (EverWinter23) [reply src/components/Card/Card.tsx]: Nope, removed. Also-- what about the type definition for `elevations`-- I've added  the type in `helpers/elevations.ts`, but not as an `interface`.

Comment (anitnilay20) [reply src/components/Card/Card.tsx]: `type` is good enough. We don need `interface`.

**Review by anitnilay20 — APPROVED** [review]

**Pull request merged**